### PR TITLE
https://issues.redhat.com/browse/ACM-14146 Fix taint command

### PR DESCRIPTION
**Pull request by oafischer**
2.12 and 2.11

### CODE REVIEW

File: clusters/cluster_lifecycle/taints_tolerations.adoc [hunk 1]
```diff
@@ -9,7 +9,7 @@ You can control the placement of your managed clusters or managed cluster sets b
 Taints are specified in the properties of a managed cluster and allow a placement to repel a managed cluster or a set of managed clusters. You can add a taint to a managed cluster by entering a command that resembles the following example:
 
 ----
-oc taint ManagedCluster <managed_cluster_name> key=value:NoSelect
+oc patch managedcluster cluster1 -p '{"spec":{"taints":[{"effect":"NoSelect","key":"cluster.open-cluster-management.io/unreachable"}]}}' --type=merge
```
Comment (haoqing0110) [inline clusters/cluster_lifecycle/taints_tolerations.adoc]: If the `taints` section doesn't exist, you can add a taint to a managed cluster by entering a command that resembles the following example:
```
oc patch managedcluster <managed_cluster_name> -p '{"spec":{"taints":[{"key": "key", "value": "value", "effect": "NoSelect"}]}}' --type=merge
```
Or append a taint to the existing taints with the following example: 
```
oc patch managedcluster <managed_cluster_name> --type='json' -p='[{"op": "add", "path": "/spec/taints/-", "value": {"key": "key", "value": "value", "effect": "NoSelect"}}]'

```

**Review by haoqing0110 — CHANGES_REQUESTED** [review]

File: clusters/cluster_lifecycle/taints_tolerations.adoc (hunk 1 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/taints_tolerations.adoc]: Updated

Comment (oafischer) [reply clusters/cluster_lifecycle/taints_tolerations.adoc]: @haoqing0110

**Review by swopebe — APPROVED** [review]

**Pull request merged**